pkgs/applications/networking/portmaster/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "portmaster";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "safing";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-XlFPM6EaoSGOV39aZW6fG9Quacw7Z4lSxDeSW2excL0=";
+  };
+
+  vendorHash = "sha256-dj+XDrcbZXdIj1WlTb5Y+v88by54t3R8zLujjtKiudA=";
+
+  # integration tests require root access
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A free and open-source application firewall that does the heavy lifting for you";
+    homepage = "https://safing.io";
+    license = licenses.agpl3;
+    maintainers = with maintainers; [ nyanbinary ];
+  };
+}
```

```suggestion
}

```

possibly that is also meant.